### DISCUSSION TITLE
【求助】我莫名奇妙的被卷入外星生物大战了怎么办啊？急急急。

### DISCUSSION THREAD

**Discussion by Jiucheng258** [share121/inter-knot#1478] [其他]
![R (1)](https://github.com/user-attachments/assets/b3f0952d-6fc0-4082-8ef6-fc09746a1910)
如你所见，我和这位小姐姐本来是在漫展上玩的，结果莫名奇妙就被外星人抓住了。。。之后又是一堆事情，好烦啊，感觉人生也就这样了，好想家。。。呜呜呜。不过我们现如今被迫降落在这颗星球上了，还被一堆外星生物包围了......虽然我们有从外星人那里抢来的科技武器，但是周围的怪物实在太多辣！能不能来个人告诉破局之法啊2333........

**Comment by 2894986830** [comment]
いや、知っていない

**Comment by lllliufeng** [comment]
敌不动，我不动（

**Comment by UmayKui3678** [comment]
死去的回忆攻击我，好早之前看中国拜玩的宅男人间冒险
